charts/n8n/values.yaml

```diff
@@ -236,54 +239,3 @@ postgresql:
   postgresqlDatabase: n8n
   persistence:
     size: "200Gi"
-
-gatekeeper:
```

Comment:
Why removing `gatekeeper` ?

Reply:
I noticed it wasn't used by the chart, so assumed it was a leftover that needed cleanup...

Reply:
Yes, We can have one separate gatekeeper chart that we install for any service, by simply overriding values.

Reply:
deploying gatekeeper via helm chart - https://open-policy-agent.github.io/gatekeeper/website/docs/install/#deploying-via-helm